modules/redhat-access-integration-backend/src/main/java/org/hawkular/accesportal/backend/control/RedHatAccessConfiguration.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.accesportal.backend.control;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import javax.enterprise.context.ApplicationScoped;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@ApplicationScoped
+public class RedHatAccessConfiguration {
+    private int sessionTimeout;
+    private boolean brokered;
+    private boolean devel;
+    private String userAgent;
+    private String url;
+    private String proxyUser;
+    private String proxyPassword;
+    private URL proxyUrl;
+    private int proxyPort;
+
+    public RedHatAccessConfiguration() throws MalformedURLException {
+        this.brokered = true;
+        this.userAgent = "redhat-access-plugin-jon-1.0.4";
```

Comment:
Could you please set userAgent to (ie. redhat-access-plugin-hawkular-1.0.4). UserAgent is used by RHA portal to generate analytics and should be in form "redhat-access-plugin-$product-$javaLibVersion"